src/RazorSdk/Tool/Microsoft.NET.Sdk.Razor.Tool.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
BOM change?

Reply:
I'd assume so. I had changed more in this file initially but removed those changes.

Reply:
Consider reverting all changes in this file then.

Reply:
I've reverted the file. You were right, it was BOM change.
![image](https://github.com/user-attachments/assets/356a2612-ba5f-47a0-ae49-6f91ac32992b)